tools/internal-rules/no-invalid-meta.js

```diff
@@ -109,15 +109,14 @@ module.exports = {
     meta: {
         docs: {
             description: "enforce correct use of `meta` property in core rules",
-            category: "Internal",
             recommended: false
         },
         type: "problem",
         schema: [],
         messages: {
             missingMeta: "Rule is missing a meta property.",
             missingMetaDocs: "Rule is missing a meta.docs property.",
-            missingMetaDocsCategory: "Rule is missing a meta.docs.category property.",
+            missingMetaDocsDescription: "Rule is missing a meta.docs.description property.",
```

Comment:
This is wrong. We don't want this `no-invalid-meta` rule to verify `meta.docs.description` anymore since this is now handled by the `eslint-plugin/require-meta-docs-description` rule. So all this description code can be removed from the rule.

Reply:
Agree. We can just remove the category check.